x-pack/plugins/security_solution/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -440,7 +440,7 @@ export const PolicyList = React.memo(() => {
                 <EmptyPolicyTable
                   loading={loading}
                   onActionClick={handleCreatePolicyClick}
-                  actionDisabled={isFetchingPackageInfo}
+                  actionDisabled={false}
                   dataTestSubj="emptyPolicyTable"
```

Comment:
I'm probably missing the context here, but it looks like we're ditching some dynamic behavior for always-not-disabled. Why?

Reply:
This component is showing a Button to create a new policy by using the `handleCreatePolicyClick` above. Now that I look closer at it, I don't think we need the `actionDisabled` prop. @kevinlog you see any issues if I remove it?

Reply:
This will actually probably not be needed anymore since we're getting the package info in a different way now.  I can clean this up further in an upcoming PR I have where I touch this stuff again

Reply:
sounds good 👍 . Thanks @kevinlog